examples/sink_id.rs

```diff
@@ -0,0 +1,29 @@
+use web_audio_api::context::{AudioContext, AudioContextOptions, BaseAudioContext};
```

Comment:
This example works best with `--features cubeb`
Cpal does not allow you to choose speaker output when a headset is plugged in